server/src/Api/Controllers/FlashcardsController.cs

```diff
@@ -53,4 +55,29 @@ public IActionResult DeleteFlashcard(int flashcardId)
         var isSuccess = _flashcardService.DeleteFlashcard(flashcardId);
         return isSuccess ? Ok() : NotFound();
     }
+
+    [HttpPost]
+    [Route("import-csv")]
+    [Consumes("multipart/form-data")]
+    public IActionResult ImportFlashcardsFromCsv(IFormFile file)
```

Comment:
Could you provide a link to reference to an example of how you made it?

Reply:
https://medium.com/@niteshsinghal85/testing-file-upload-with-swagger-in-asp-net-core-90269bc24fe8